source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -831,3 +852,117 @@ Name           | Type   | Value | Description
   }
 }
 ~~~
+#### Example 8 (pickup from collection point selected)
```

Comment:
Leave an empty line before the heading:

![image](https://user-images.githubusercontent.com/1265295/87933282-a5463100-ca95-11ea-903c-e6f406552713.png)